beslab/0.0.1/besman-private-artifact-repo.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+function __besman_install_jfrog()
+{
+    echo "Installing jfrog"
```

Comment:
Can we add Todo here? seems function is yet to develop right ?

Reply:
Done